main/res/layout/dialog_edittext.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
One more thing: I don't think you have to wrap the `TextInputLayout` in another layout here, you might have less problems with programmatic view creation if you just put the `TextInputLayout` here as top-level view (like e.g. done with the `MaterialButton` in `button_view.xml` or `button_icon_view.xml`). Also, such a view snippet is more flexible usabe by other parts of the code later.

Reply:
Ah, good idea, thanks! I'm so used to having either LinearLayout, FrameLayout or something similar as enclosing frame that I did not try TextInputLayout yet. Will do so, would make things easier.